src/App.tsx

```diff
@@ -6,17 +6,27 @@ import { Projects } from '@pages/Projects/Projects';
 import { Registration } from '@pages/Registration';
 import { apiClient } from '@utils';
 import { Route, Routes } from 'react-router-dom';
+import { useTicketStore } from './store/useTicketStore';
 import { useUserStore } from './store/useUserStore';
+import { getUserTicket } from './utils/controller/ticket.controller';
 
 function App() {
   const setUser = useUserStore((state) => state.setUser);
+  const setTicket = useTicketStore((state) => state.setTicket);
 
   useEffect(() => {
     apiClient.auth.onAuthStateChange((_event, session) => {
       if (!session?.user) return;
```

Comment:
`onAuthStateChange` dispatch callback on several occasions, if you put console.log inside, when the tab resumes focus the event is fired.

https://supabase.com/docs/reference/javascript/auth-onauthstatechange

Maybe we have to condition the type of event so that it is only fired in the case that we require (maybe INITIAL_SESSION ?)